client/graphite/read.go

```diff
@@ -192,12 +193,27 @@ func (c *Client) targetToTimeseries(ctx context.Context, target string, from str
 				"path", renderResponse.Target, "prefix", c.cfg.DefaultPrefix, "err", err)
 			return nil, err
 		}
-		for _, datapoint := range renderResponse.Datapoints {
-			timstampMs := datapoint.Timestamp * 1000
+		for j, datapoint := range renderResponse.Datapoints {
+			timestampMs := datapoint.Timestamp * 1000
 			if datapoint.Value == nil {
 				continue
 			}
-			ts.Samples = append(ts.Samples, &prompb.Sample{Value: *datapoint.Value, Timestamp: timstampMs})
+			ts.Samples = append(ts.Samples, &prompb.Sample{Value: *datapoint.Value, Timestamp: timestampMs})
```

Comment:
can you put that in another function to avoid going too much to the right ?

Reply:
Done.